packages/tabs/src/utils/getSelectedIndex/getSelectedIndex.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * @param selected
+ * @param tabTitleElements
+ * @returns Returns the corresponding index for a selected string
+ */
+export const getSelectedIndex = (
+  selected: number | string,
+  tabTitleElements: Array<HTMLElement>,
+) => {
+  if (typeof selected === 'number') return selected;
+
+  return tabTitleElements.findIndex(
```

Comment:
Also, what happens if we don't find anything?

Reply:
It's the same as passing an index that doesn't exist—nothing will be selected. I can document this.